### DISCUSSION TITLE
How often do you release new version to nuget ?

### DISCUSSION THREAD

**Discussion by psulek** [riok/mapperly#192] [General]
As i completed my 1st PR in this repo, im curious when my changes can see the lights in form of new version in nuget.org ?

**Comment by latonz** [comment]
We'll probably await https://github.com/riok/mapperly/pull/181 and create a preview release as soon as https://github.com/riok/mapperly/pull/181 is merged.

**Reply by psulek** [reply]
sounds good!

**Reply by latonz** [reply]
[2.6.0-next.1](https://www.nuget.org/packages/Riok.Mapperly/2.6.0-next.1) is released now.
